projects/angular/src/accordion/_accordion.clarity.scss

```diff
@@ -7,160 +7,70 @@
 @use 'sass:meta';
 @use '../utils/a11y';
 @use '../utils/mixins';
-@use '../utils/variables/variables.global';
+@use '@cds/core/tokens/tokens.scss';
 @use 'variables.accordion' as accordion-variables;
 @include meta.load-css('properties.accordion');
 
 @include mixins.exports('accordion.clarity') {
   .clr-accordion {
     display: block;
-    margin-bottom: variables.$clr_baselineRem_1;
+    margin-bottom: tokens.$cds-global-space-9;
   }
 
   .clr-accordion-panel {
     display: block;
   }
 
   .clr-accordion-header {
-    $box-shadow-default: inset accordion-variables.$clr-accordion-border-left-width 0 0
-      accordion-variables.$clr-accordion-border-left-color;
-    @include mixins.css-var(
-      color,
-      clr-accordion-text-color,
-      accordion-variables.$clr-accordion-text-color,
-      variables.$clr-use-custom-properties
-    );
-    border: variables.$clr_baselineRem_1px solid;
-    @include mixins.css-var(
-      border-color,
-      clr-accordion-border-color,
-      accordion-variables.$clr-accordion-border-color,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      box-shadow,
-      clr-accordion-header-left-indicator,
-      $box-shadow-default,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      background,
-      clr-accordion-header-background-color,
-      accordion-variables.$clr-accordion-header-background-color,
-      variables.$clr-use-custom-properties
-    );
+    color: accordion-variables.$clr-accordion-text-color;
+    box-shadow: accordion-variables.$clr-accordion-header-left-indicator;
+    background: accordion-variables.$clr-accordion-header-background-color;
     transition: all 200ms ease-in-out;
-    border-bottom: 0;
+    border: tokens.$cds-global-space-1 solid accordion-variables.$clr-accordion-border-color;
+    border-bottom-width: 0;
     width: 100%;
-    @include mixins.css-var(
-      font-size,
-      clr-accordion-header-font-size,
-      variables.$clr_baselineRem_0_65,
-      variables.$clr-use-custom-properties
-    );
```

Comment:
I see that property was exposed before TM so you will need to use it as override in the mixin or deprecate it.

Reply:
Added "Deprecated" in STYLES.md and "@Deprecated" in the variables file